manifests/init.pp

```diff
@@ -92,12 +92,9 @@
     fail('cfg parameter must be a hash')
   }
 
+  anchor { 'grafana::begin': } ->
```

Comment:
Yes, but there is an issue regarding broken dependencies. According to `metadata.json` Puppet >= 3.2 is supported, but `contains` was added at Puppet 3.4. The issuer could not use this module anymore, because he still uses Puppet 3.2.

I think this will be fixed with a lot of other stuff during the migration to Puppet 4.